### ISSUE TITLE
DX12: ID3D12GraphicsCommandList exception

### ISSUE THREAD

**Issue by xiaopengli89** [gfx-rs/wgpu#3372]
**Description**
```
wgpu_hal::auxil::dxgi::exception - ID3D12GraphicsCommandList::*: This API cannot be called on a closed command list. [ EXECUTION ERROR #547: COMMAND_LIST_CLOSED]
```

**Platform**

OS: Win10
CPU: Intel(R) Core(TM) i3-8109U CPU @ 3.00GHz
GPU: Intel(R) Iris(R) Plus Graphics 655

**Comment by teoxoy** [comment]
Is it happening in one of our tests?
If not, could you send us a test case so that we can reproduce it?
Thank you!

**Comment by xiaopengli89** [comment]
Fixed by #3204 .
